feature-libs/product-configurator/rulebased/core/connectors/rulebased-configurator.adapter.ts

```diff
@@ -112,13 +112,16 @@ export abstract class RulebasedConfiguratorAdapter {
   ): Observable<Configurator.Overview>;
 
   /**
-   * Abstract method to get configuration overview
+   * Abstract method to get the supported configurator type
    *
-   * @param configId configuration id
-   * @param owner configuration owner
    */
   abstract getConfiguratorType(): string;
 
+  /**
+   * Abstract method to check if the adapter works only with OCC?
+   */
+  isOccOnly?(): boolean;
```

Comment:
In the comment above the method you wrote `abstract method` but before the method name you did not use `abstract` keyword